src/components/adaptive-cards/InputChoiceSet/InputChoiceSet.jsx

```diff
@@ -16,18 +17,20 @@ import {formatDateTime} from '../util';
  * @param {object} props  React props passed to the component
  * @param {object} props.data  Active cards definition
  * @param {string} [props.className]  Custom CSS class to apply
+ * @param {string} [props.id]  InputChoiceSet id
```

Comment:
no need to list the component name here (we are not doing it for className or other props).

just "DOM id" should be fine